SonarQube.TeamBuild.PostProcessor/Program.cs

```diff
@@ -27,9 +27,18 @@ private static int Main(string[] args)
 
             AnalysisConfig config = GetAnalysisConfig(settings, logger);
 
-            MSBuildPostProcessor postProcessor = new MSBuildPostProcessor(new CoverageReportProcessor(), new SonarRunnerWrapper(), new SummaryReportBuilder());
+            bool succeeded;
+            if (config == null)
+            {
+                succeeded = false;
+            }
+            else
+            {
+                MSBuildPostProcessor postProcessor = new MSBuildPostProcessor(new CoverageReportProcessor(), new SonarRunnerWrapper(), new SummaryReportBuilder());
+
+                succeeded = postProcessor.Execute(args, config, settings, logger);
+            }
```

Comment:
Miscellaneous: GetAnalysisConfig will already have logged the missing config so don't call the post-processor.